src/raftstore/store/worker/apply.rs

```diff
@@ -1293,6 +1293,9 @@ impl Runner {
         let region_id = s.region.get_id();
         let term = s.term;
         let delegate = ApplyDelegate::from_registration(self.db.clone(), s);
+        info!("{} register to apply delegates at term {}",
+              delegate.tag,
+              delegate.term);
```

Comment:
Seem not necessary here. Outer always logs the region info before registration. 

@hhkbp2